docs/overview/uninstall_inductiva.md

```diff
@@ -0,0 +1,17 @@
+# Uninstall the `inductiva` package
+
+Whether you just want a completely new installation of `inductiva` or
+whether you decided that, as it turns out, the package is not for you,
+we understand and would like to thank you for having given it a
+go. Either way we would like to give you instructions on how to
+completely remove traces of `inductiva` from your system.
+
+First you should remove it using:
+
+```shell
+pip uninstall inductiva
+```
+
+Next, recall that the inductiva package, writes locally to your
+disc. Therefore you should have a folder under `$HOME/.inductiva` in
+your system. Remove it. You are good to go!
```

Comment:
````suggestion
disc. Therefore, you should have a folder under `$HOME/.inductiva` in
your system. Remove it and you are good to go!

```shell
rm -rf $HOME/.inductiva
```
````

Reply:
Use *disk instead, since that refers more to a computational sense and what we use as arg in the library.